.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+Thank you for your contribution!
+
+Please provide a summary of the changes and the motivation behind the change.
+
+## Reviewer Checklist
+
+- [ ] Documentation
```

Comment:
This seems a bit non-actionable at the moment, right? If we want commiters to write docs, we probably first need to boostrap some docs and write something about what requires docs, where to put them, etc.

Reply:
I can create a `/docs` dir in this commit, do we really need a fully fleshed out doc with examples?

Reply:
Having a designated spot and allowing someone to add to it when appropriate seems reasonable

Reply:
I think some structure or at least some guidelines about structure is important too. Otherwise, we'll end up with a whole bunch of stuff duplicated or in random-ish places. I say this from experience with rukpak docs evolution.

Maybe we start with something that talks about ensuring Go Docs exist for the CRD API fields. I honestly feel like we would get a huge amount of mileage if that's _all_ we documented, for now/starting out.

We can simultaneously figure out a docs strategy around OLMv1, but I feel like that deserves more planning and attention.

Reply:
Removed the `getting-started doc` commit.

Reply:
Updated the documentation check for Go Doc exists for CRD api.